### PR TITLE
Nerfs 762, 556, midlasers, specialised armour, personal shields and stims.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
The idea is to make combat take longer and not be as particularly onesided, with less reliance on items that are frustrating to fight against. Currently, munitions like 762/556 are extremely powerful and dominate combat, with one-two shots deciding most engagements.

Nerfing the damage dealt by these bullets also requires nerfing specialised armour (ballistics/ablatives) because otherwise they'd outright block everything. Thus, specialised armour now blocks 5 to 10 more damage when compared to heavy armour values.

Example values:
```
7.62 (35dmg 22ap) 
vs heavy armour (50) = 21.8
vs ballistic (55) = 18.5
```
```
5.56 (30dmg 26ap) 
vs heavy armour (50) = 18.7
vs ballistic (55) = 15.5
```
```
5.56 lethal polymer (24dmg 32ap) 
vs heavy armour (50) = 16
vs ballistic (55) = 12.5
```
```
shotgun slug (55dmg 0ap)
vs heavy armour (50) = 30.25
vs ballistic (55) = 19.25
```
```
midlaser (30dmg 14ap)
vs heavy armour (40) = 17.3
vs ablative (45) = 10.1
```
As a result, personal shields and stims were also nerfed, since they aren't as necessary now that weapons don't instakill.

- Combat hypo chems have been compacted into one chem (bar oxycomorphine which is no longer included), Kilosemine, which metabolizes at a rate of 0.6u per tick.
- Personal shields now have a battery. Passive upkeep is 50 charge, blocking a shot costs 250. They can be recharged in rechargers and the battery cannot be removed or replaced manually.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

**Review by SleepyGemmy — APPROVED** [review]

**Comment by TheDanseMacabre** [comment]
This is an extremely drastic and regressive PR that brings us back to the time when rifles did pistol-tier damage. I really don't understand where this PR is coming from, but I can tell you that it's essentially going to increase survivability to the point that anyone is capable of walking off anything they want - it'll end up feeling like we're playing /tg/. Making rifles do pathetic damage because they're strong is not a good idea. If anything, it'd be better to find some way to limit their use or incentivize using weapons other than rifles.

**Comment by Zelmana** [comment]
Personal shields break after 4 uses already, no? What's the deal with this charge stuff? It's a throwaway item as-is.

**Comment by SleepyGemmy** [comment]
> Personal shields break after 4 uses already, no? What's the deal with this charge stuff? It's a throwaway item as-is.

personal shields recharge once per minute and reset to max charges, as far as i know.

**Comment by NonQueueingMatt** [comment]
> Personal shields break after 4 uses already, no? What's the deal with this charge stuff? It's a throwaway item as-is.

No, without this PR they recharge their uses after 1 minute (and block 5 shots)

File: html/changelogs/mattatlas-alikesupreme.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - balance: "Damage values of 7.62, 5.56 and midlasers have been all nerfed."
+  - balance: "Security's ballistic and ablative armour have been heavily nerfed. They now only block around 5 more damage, instead of blocking over half."
+  - balance: "Combat stims' chems have been compacted into one new chem, Kilosemine, with increased metabolisation rate (0.6u/tick) and without the effects of Oxycomorphine. Combat hyposprays start with ten units, which can be spread out across two injections."
+  - balance: "Personal shields no longer recharge on their own. They have a cell, which can be recharged and cannot be manually replaced. Passive upkeep is 50 charge per tick, whereas blocking a bullet is 200 charge per tick. Their size has been reduced to normal, so they can fit in backpacks."
```
Comment (Arrow768) [inline html/changelogs/mattatlas-alikesupreme.yml:44]: That is not correct.
Passive Upkeep (currently) is 50 per 2 seconds.
Active Upkeep is not 250 per 2 seconds or tick, but 250 per blocked shot.

**Review by Arrow768 — DISMISSED** [review]

**Review by Alberyk — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]

**Pull request merged**